### ISSUE TITLE
Multiple inaccuracies/flaws in chocobo digging system.

### ISSUE THREAD

**Issue by Ladypinky** [EdenServer/community#2548]
I would like to say I have enjoyed that Eden has chocobo digging at all! However I've noticed a plethora of inaccuracies from Eden/Retail. I just want to address everything I know from being a Veteran Pre-fatigue retail chocobo digger.

Firstly, in retail items spawn every 1:15 game hours, not every 1:00. When ever conquest/daily tally posts is was an easy way to know when next items were added because that messaged was in sync with chocobo digging. <- that's how you know I'm legit :) (fun fact, you could dig at x:13 to still scoop up an item at X:15<- how you know I'm more legit lol).

Secondly, digging rank affects chance to dig up items based on rarity on retail, which is COMPLETELY missing on Eden. A lot of people have the misnomer that ranking up increases your accuracy, which is true, but it is more or less your chance to dig up rare tiers of items(e.g. rank0 would have 1% chance for Orichalcum ore, whereas Rank10 would have 90%+ or higher on Orichalcum ore). I remember in retail around Rank6, I would dig up Rolanberry fields until it was barebones dry, and even then it would take me 20+ digs of finding nothing until I would finally pull up the Orichalcum ore and then I'd continue to find nothing until items spawned(or possibly another ori ore if a 2nd was hiding lol). On Eden I obtained my first Orichalcum ore at Rank 1 in a fresh zone. Without a rarity tier system, there is no reason to rank up other than timers and the tiny bit of accuracy it seems to give. Of course I do not have exact numbers but I do know this rarity system exists. I remember Orichalcum ore not really being possible until rank 8. You could find them anytime, just your chance to successfully dig it up was abysmal, so you get the "find nothing" message if it was the only item left in zone. It was like certain ranks were milstones for finding certain items. You could find anything at any rank but it felt like adman ore for instance maybe you would have 10% chance to get if it was only item in zone at rank7 but at rank8 you'd have 40% chance in that same scenario. ***Also I'm not entirely sure how loot works on Eden but in retail items spawned into a pool based on % chance. Meaning once an item was in the zone, it sat there until dug up. The system worked more like this: if you had Sage, Flintstone, Orichalcum ore in the zone, you're chance to pick anyone of those items was 33%, but your accuracy for each was different. So let's say you're rank0 and you're 33% draws the Orichalcum ore, well now you're going to either have a 1% chance to dig it up or find nothing. <- that in essence is where your rank comes into play, its nice to have lower start/delay times but Rank is more about power. (But also items had caps, there could be no more than 2 orichalcum ore in rolanberry fields at max, if you found 2, then getting a 3rd would be extremely lucky and only possible if it spawned at next spawn time. <-- this is what keeps balance. On Eden this system doesn't exist AFAIK because sometimes I can get 5+ elemental ores in a single zone, that would be UNREAL on retail, I think 3 is the most I've found in 1 zone during 1 game hour(assuming 2 were already spawned, and 3rd was just spawned). On retail we dug Elemental ores differently than we do here on Eden. I do not know which system is better because they both produce roughly the same amount of ele ores per digger, getting 10-14 ele ores per 8 hours is pretty retail accurate(Prefatigue) but the strategy to get them is different on Eden. I'm guessing Eden's chocobo digging is just a series of dice rolls, which is similar but then you can get as many ores as you have dicerolls remaining. On retail we'd move zone to zone, if you got an ele ore, you wouldn't dare try for an ore in that zone for at least 30 minutes because there was probably another zone out there with 1-2 already sitting in it. Also I think retail ele ore's persisted through loss of weather, kind of like Sirocco Kukri NM, this is why we'd find most our ele ores in the early parts of ele ore digging, and less as time went on. After 3-4 hours most JP diggers would stop going for ele ores because you'd be getting like 1-2 per hour instead of 3-5. (Granted on my server there were at least 20 active JP diggers, 2-3 RMT bots that dug 24/7, and 1 other NA max rank).

Because of these rarity tiers, it was tradition that you start chocobo digging in the jungles, even the highest tiers of logs were not considered all that rare by the digging system. Only Kingtruffle/Reishi mushroom were hard to dig up as they're on the same tier as Ele ores in difficulty-fun fact an elemental ore is about as about as difficult as digging up a philosopher's stone on retail. Also mainlands seemed to have penalty on accuracy like the moon phases 45%-60% for lower level diggers, it wasn't worthwhile to even attempt mainland digging until you were rank8~. I remember by accuracy at max rank being through the roof in retail. Like 90% regardless of any moon/mainlands. 

Here is a short video of a digger during the worst moon phase in west desert.
https://www.youtube.com/watch?v=tOcZoHNPuBg
My accuracy at max rank isn't even that high on full/new moon on Eden...

Thirdly, accuracy rates on Eden are glitched!
I started to dig on my alt see do more tests and have realized how low/glitched accuracy is.
I frequently get a bug where I miss 10-20 in a row on my max rank digger, while my rank5 alt just casually scoops up items until it too hits a wall of missing 10-20 times, mind you this is in the same zone, so I know there are still items in the pool. I'd say it was just bad luck, but this happens so frequently that it seems accuracy is bugged, or rather that accuracy is just still too low at higher ranks. On retail you would /fume if you missed 5-6+ digs, that's how I feel at max rank missing 15+. It just feels really off/glitchy.

Conclusion
I believe digging needs a second look. I don't expect much to be changed about the current system. I think accuracy rates for all ranks need a boost, especially rank 7 and up by at least an additional 5% per rank and maybe 10% from 9 to 10. I realize that everyone seems to think digging on Eden is over powered, but you guys have no idea what it was like on retail, digging on Eden is a HEADACHE sometimes lol! Retail a max rank digger can clear a fresh zone in 12-16 stacks of greens, on Eden it takes a Rank10 50 stacks of greens to barely dig up 1 zone. So it doesn't just cost more greens, but it also costs more time, for barely anymore rare items. I really want my rank to actually mean something more than just having lower timers. I'm tired of being max rank with 50% accuracy when I'm use to having at least 90% in retail. I know the general consensus is that digging needs a nerf, but realize it's becoming more popular and for every 1 max rank digger it lowers every other max rank diggers profits by 50%!!!!!!!!!! If anything we need more buffs, especially with all the new people digging out there!

Throwing this in at the end too for additional resources. Here's Ayrlie, a 2006 A20(Rank8) Retail digger finding an elemental ore. https://www.youtube.com/watch?v=JHs1-s1aX_o

Keep in mind how few times shes missing digs DESPITE there being an ele ore hiding in the zone! Her accuracy is about how mine is at Rank10, but I'm also occasionally finding nothing 10-20 times in a row every couple of minutes.

**TL:DR-** Accuracy rates suck on Eden, digging needs a boost for all ranks. And there is no need to nerf something that is already nerfed into the ground. Please realize that digging will nerf it's self as more diggers start to dig, less profit will be made by all. The only way to balance digging is to have more players dig, but that won't happen because accuracy rates suck and only digging ele ore is worthwhile. People think ele ores are OP, but if there's 2-3 other people digging I go from finding 10-15 in 8 hours, to 3-5 in 8 hours. That's brutal enough.

Thank you for taking these issues into consideration! I look forward to the issue being looked at!

- Ladypinky

**Comment by KnowOne134** [comment]
Very nice report. Thanks for sharing

**Comment by ZariusCane** [comment]
Just want to add onto this.  I have a sample size of digging that was about as controlled as it could possibly get on Eden.  I have an extensive list of known diggers. During this recording, I dug in less popular zones and when a known digger arrived, I left the zone. When the zone was dug out, I stopped and waited for the pool to refresh.  This was at adept rank.

![digging](https://user-images.githubusercontent.com/16099481/88536934-39eaea80-cfd2-11ea-8d51-9184b033ac9b.jpg)

according to the wiki, accuracy should be really high at high levels.

![digger2](https://user-images.githubusercontent.com/16099481/88537038-67379880-cfd2-11ea-8c77-308e70814a8e.png)

**Comment by Ladypinky** [comment]
I think the digging system on private servers in general needs a rework from the ground up. It appears loot spawns based on a weighted system but it doesn't have slots to hold items. If you dig an orichalcum ore or elemental ore within the first few digs of a zone and that's the only item you want to find, you can't simply stop digging there like on retail. You must clear the entire zone or it'll never spawn because it doesn't have a slot to spawn in. If a zone is full, what ever loot is in it~ is in it.  I think that is probably the biggest annoyance with digging. Digging high ranks isn't about spamming gysahl greens, it's about being able to find the rare items with less greens~ you still have to use a fair amount, which will never be true on Eden because items don't have difficulty ranks. "Test items" on retail was a system JPs used to show the power of digging. You wouldn't start seeing Reishi mushrooms, Orichalcum, and adaman ore until you were like rank 8, but probably 9+. So if you hit rank 4, your JP friend might have told you "oh, you'll start seeing Rosewood logs more commonly now!" Rank1 would take 200~300 stacks of greens to find 1 orichalcum ore, rank10 would take 3-6 stacks to find 1-2 orich ores- if it was even in the zone at all.

I think the biggest problem with digging on Eden in general is the bore/burrow loot isn't on a separate table. In retail if you had bore/burrow you could obtain items from their respective tables. On Eden if you don't have bore/burrow the game bugs out and if you dig a bore or burrow item its a "find nothing". This is easy to test and to prove~ Find a fresh zone and the first 50 or so digs, it'll seem like the zone is dry. This is because you're digging up a fair amount of bore/burrow loot. Bore/burrow loot holds less weight so when it's mostly gone, the problem clears up. However, it makes a zone take 30~40 stacks of greens to clear rather than 15~20 stacks. Right now you spend like 15k to make 40k per rank 10 digger. Either the price of materials must go up, gyashl green costs must go down, or the problem should be looked into.

This all probably sounds "too good to be true" but I remember the true savagery that chocobo digging was. The impossibility of digging mainland zones for profit until you were rank 8+ due to item difficulty, deserts being fought over to the death 24/7, and RMT realizing digging was the ultimate secret of FFXI. Just as not everyone was intended to be 100 goldsmithing, not everyone was intended to even have knowledge about chocobo digging. Knowing chocobo digging was like being a scientist at Area 51. It was above top secret. You have to realize the insanity it took to sit there and chocobo dig for hours to get the first rank up. You have to put a lot of belief and faith in it because the best item you'd probably find at rank0 would be a danceshroom or maybe a rosewood log after dozens of stacks of greens. The only zone you could dig at rank0 was probably Yuhtunga jungle, even starter zones were suicide because the difficulty of mainlands was so much higher than jungles.

TL;DR~ Add bore/burrow loot to a seperate loot table requiring bore/burrow to be accessed and maybe consider adding a general difficulty system for at least 4(common/uncommon/rare/ultra rare) tiers of loot~ Rank10 digging for the most part feels pretty era accurate other than needing to constantly dig up zones to force spawn rare loot to spawn. It's still more fun than vendor crafting shihei~ just not as good of gil per hour.

**Comment by MB213** [comment]
_"On Eden if you don't have bore/burrow the game bugs out and if you dig a bore or burrow item its a "find nothing". This is easy to test and to prove~ Find a fresh zone and the first 50 or so digs, it'll seem like the zone is dry. This is because you're digging up a fair amount of bore/burrow loot. Bore/burrow loot holds less weight so when it's mostly gone, the problem clears up."_

When I first enter a zone it often acts like it's been dug out. I can dig for 5~7 minutes without getting much of anything until finally I start finding items and it acts normally for the rest of my digging session. This is for sure a trend; I used to leave zones early thinking they were dug out. It took me a while to realise that you need to persist through that first phase. I do not know at all what the cause of this is, I am unaware whether burrow and bore tables are even coded, but I agree with Ladypinky that this does happen.

**Comment by ZariusCane** [comment]
Elemental Ore currently can be dug on both waning and waxing moons at 7-24% on Eden.  This explains why there is so much ele ore in the market.  According to the wiki from 2007 history, Ele ore should only be dug during 7-24% waxing moon.  

![waning](https://user-images.githubusercontent.com/16099481/92866794-8db86400-f3c5-11ea-8e51-b17326cf04cf.png)

![img_20200624_140420](https://user-images.githubusercontent.com/16099481/92867487-4ed6de00-f3c6-11ea-8bbc-025ba24d25a1.jpg)

![img_20200624_140642](https://user-images.githubusercontent.com/16099481/92867529-59917300-f3c6-11ea-84ca-38aeed063d82.jpg)

There was a few minutes from the time stamp, not the best pictures but worthy of investigation.

**Comment by ffxi-waxer** [comment]
![edenxi_iL3E7p0i2j](https://user-images.githubusercontent.com/71594660/93764662-59645500-fbc8-11ea-8365-5388f09cc16d.png)

I think we can summarize just how bad digging is at the moment by comparing it to crafting.

Imagine you leveled your goldsmithing to 100 only to reduce the crafting time to 0 and receive zero bonuses to HQ chance. That's basically rank 10 chocobo digging at the moment. Outside of rank 7 allowing elemental ores to be obtained, rank just speeds up how fast you go through your greens. 1% orichalcum at rank 0 and rank 10 is 1%

Bore/Borrow items need to either be removed from the database or changed to no requirement until they're either obtainable or added to their own loot pool. The amount of missed digs that come from Bore/Borrow items makes digging accuracy way below what it should be.
